ivy/data_classes/container/experimental/activations.py

```diff
@@ -1187,3 +1187,115 @@ def tanhshrink(
+        to_apply: Union[bool, ivy.Container] = False,
+        prune_unapplied: Union[bool, ivy.Container] = True,
+        map_sequences: Union[bool, ivy.Container] = False,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.softshrink. This method simply wraps
+        the function, and so the docstring for ivy.softshrink also applies to this
+        method with minimal changes.
+
+        Parameters
+        ----------
+        x
+            input container.
+        lambd
+            Lambda value for soft shrinkage calculation. Default is ``0.5``.
+
+        Returns
+        -------
+        ret
+            Container with soft shrinkage applied to the leaves.
+
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([1., -2.]), b=ivy.array([0.4, -0.2]))
+        >>> y = ivy.Container._static_softshrink(x)
+        >>> print(y)
+        {
+            a: ivy.array([0.5, -1.5]),
+            b: ivy.array([0., 0.])
+        }
+        """
+        return ContainerBase.cont_multi_map_in_function(
+            "softshrink",
+            x,
+            lambd=lambd,
+            key_chains=key_chains,
+            to_apply=to_apply,
+            prune_unapplied=prune_unapplied,
+            map_sequences=map_sequences,
+            out=out,
+        )
+
+    def softshrink(
+        x: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+        /,
+        *,
+        lambd: ivy.Container = 0.5,
+        key_chains: Optional[Union[List[str], Dict[str, str], ivy.Container]] = None,
+        to_apply: Union[bool, ivy.Container] = False,
+        prune_unapplied: Union[bool, ivy.Container] = True,
+        map_sequences: Union[bool, ivy.Container] = False,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """
+        Apply the soft shrinkage function element-wise.
+
+        Parameters
+        ----------
+        x : array
+            Input container.
+        lambd : float, optional
+            Lambda value for soft shrinkage calculation. Default is ``0.5``.
+        key_chains : sequence of ints, optional
+            The key-chains to apply or not apply the method to. Default is ``None``.
+        to_apply : bool or sequence of bools, optional
+            If True, the method will be applied to key_chains, otherwise key_chains
+            will be skipped. Default is ``False``.
+        prune_unapplied : bool or sequence of bools, optional
+            Whether to prune key_chains for which the function was not applied.
+            Default is ``True``.
+        map_sequences : bool or sequence of bools, optional
+            Whether to also map method to sequences (lists, tuples).
+            Default is ``False``.
+        out : array, optional
+            optional output container, for writing the result to. It must have a shape
+            that the inputs broadcast to.
```

Comment:
Please remove all type hints, they are being added automatically by our doc builder

Reply:
You also may remove defaults

Reply:
mode the all changes you suggested.